lms/templates/dashboard/_dashboard_status_verification.html

```diff
@@ -1,25 +1,29 @@
 <%page expression_filter="h"/>
 <%namespace name='static' file='../static_content.html'/>
 <%!
-from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
 %>
 
 %if verification_status == 'approved':
     <li class="status status-verification is-accepted">
         <span class="title status-title">${_("Current Verification Status: Approved")}</span>
         <p class="status-note">${_("Your edX verification has been approved. Your verification is effective for one year after submission.")}</p>
     </li>
-%endif
-
-%if verification_status == 'pending':
+%elif verification_status == 'pending':
     <li class="status status-verification is-pending">
         <span class="title status-title">${_("Current Verification Status: Pending")}</span>
         <p class="status-note">${_("Your edX ID verification is pending. Your verification information has been submitted and will be reviewed shortly.")}</p>
     </li>
-%endif
-
-%if verification_status in ['must_reverify', 'expired']:
+%elif verification_status in ['denied','must_reverify', 'must_retry']:
+    <li class="status status-verification is-denied">
+        <span class="title status-title">${_("Current Verification Status: Denied")}</span>
+        <p class="status-note">${_("Your verification data was not accepted. To receive a verified certificate, you must re-submit a new photo of yourself and your government-issued photo ID before the verification deadline for your course.")}</p>
```

Comment:
Nit: I'd say "Your verification submission was not accepted.", since data isn't something all learner's would understand.